index.html

```diff
@@ -0,0 +1,137 @@
+        <h3 class="coral">Mindset Mentoring</h3>
+        <p>Build the best mindset for success - a “growth mindset” - by cultivating perseverance,
+           self-reflection, and positive decision-making with our mentors.</p>
+      </article>
+      <article id="exclusive-content">
+        <img src="./icons/icon-content-coral.png" alt="Exclusive content icon">
+        <h3 class="coral">Exclusive Content</h3>
+        <p>Get lifetime access to TIACE's digital library with exclusive study materials 
+          and bonus videos to enhance the impact of the Architecture Bootcamp.</p>
+      </article>
+      <article id="forum">
+        <img src="./icons/icon-forum-coral.png" alt="Forum icon">
+        <h3 class="coral">Forum</h3>
+        <p>Have access online community focused on architecture, development, and planning. 
+          Network with a world-class community of peers, experts and industry leaders.</p>
+      </article>
+      <a href="#" class="see-program">SEE THE WHOLE PROGRAM</a>
+      <a href="#" class="join-tiace">Join TIACE Architecture<br> Bootcamp 2023</a>
+    </section>
+    <section id="coaches">
+      <h2>Featured Coaches</h2>
+      <hr class="coral">
+    </section>
+    <button id="more" type="button" class="latto">MORE<img src="./icons/down-arrow.png" alt="Arrow down"></button>
+    <section id="partners" class="hidden">
+      <h2>Partner</h2>
+      <hr>
+      <!-- <ul class="partners-list">
+        <li><a href="https://it-park.uz/"><img src="./icons/partner-itpark-grey.png"></a></li>
+        <li><a href="https://www.instagram.com/academic_uz/"><img src="./icons/partner-academic-grey.png"></a></li>
+        <li><a href="https://tcibc.uz/"><img src="./icons/partner-tashkent-city-grey.png"></a></li>
+        <li><a href="https://newuu.uz/en"><img src="./icons/partner-new-uzbekistan-grey.png"></a></li>
+        <li><a href="https://mc.uz/?lang=ru"><img src="./icons/partner-architecture-ministry-grey.png"></a></li>
+      </ul> -->
+      <ul class="partners-list">
+        <li><a href="https://tcibc.uz/">TASHKENT CITY</a></li>
+        <li><a href="https://www.instagram.com/academic_uz/">ACADEMIC</a></li>
+        <li><a href="https://it-park.uz/">IT PARK</a></li>
+        <li><a href="https://newuu.uz/en">NEW UZBEKISTAN UNIVERSITY</a></li>
+        <li><a href="https://mc.uz/?lang=ru">Ministry of Construction<br> of the Republic of Uzbekistan</a></li>
+      </ul>
+    </section>
+  </main>
+  <footer class="latto hidden">
+    <img class="footer-desktop-img" src="./icons/logo3.png" alt="Logo">
+    <div>
+      <p>@ 2024 Tashkent Institute of Architecture and Civil Engineering. All Rights Reserved.</p>
+      <p class="privacy-policy hidden">For details about how we use your information, please see our <a href="#">Privacy Policy</a>.</p>
+      <p class="terms-cookies hidden"><a href="#">Terms & Conditions</a> | <a href="#">Cookies</a></p>
+    </div>
+  </footer>
```

Comment:
I would recommend aligning the footer in the center just like in the provided design. 

| Your design  | Provided design |
| ------------- | ------------- |
| ![image](https://github.com/Zilola-Nazarova/capstone-project-module-1/assets/33268927/3955b0dd-fecf-47fa-88f5-446c7036cccb) | ![image](https://github.com/Zilola-Nazarova/capstone-project-module-1/assets/33268927/e5f39d25-85ea-413f-aa00-d6844c3ad48f) |